media/css/firefox/new.less

```diff
@@ -341,7 +341,8 @@ html[lang|="en"] #main-headline span:first-child {
     width: 100%;
 }
 
-.download-button-wrapper.android {
+.download-button-wrapper.android,
+.download-button-wrapper.ios {
```

Comment:
Along these lines, do we need to alter the display for users on Fx iOS, like we do for users on Fx Android?